### PR TITLE
Implemented fix for GitHub Issues activity errors and resolved target/build/deploy/package issues

### LINKED ISSUE #63: GitHub Issues metrics are no longer working
In review of the scheduled task logs, I ran across the following.  I am wondering if this is related to the XML serialization issue introduced in DNN 9.13.1.  🤔 

@valadas if this is related to the mentioned DNN issue, perhaps we can look into this one together and see if we can find some clues on how to resolve this in DNN better for third-party modules like this one.

CC: @WillStrohl @bdukes @mitchelsellers @donker @mtrutledge

> Activity: GitHub Issues
System.AggregateException: One or more errors occurred. ---> System.OverflowException: Value was either too large or too small for an Int32. at System.Convert.ToInt32(Int64 value) at System.Int64.System.IConvertible.ToInt32(IFormatProvider provider) at System.Convert.ChangeType(Object value, Type conversionType, IFormatProvider provider) at Octokit.PocoJsonSerializerStrategy.DeserializeObject(Object value, Type type) in C:\projects\octokit-net\Octokit\SimpleJson.cs:line 1442 at Octokit.Internal.SimpleJsonSerializer.GitHubSerializerStrategy.DeserializeObject(Object value, Type type) in C:\projects\octokit-net\Octokit\Http\SimpleJsonSerializer.cs:line 193 at Octokit.PocoJsonSerializerStrategy.DeserializeObject(Object value, Type type) in C:\projects\octokit-net\Octokit\SimpleJson.cs:line 1495 at Octokit.Internal.SimpleJsonSerializer.GitHubSerializerStrategy.DeserializeObject(Object value, Type type) in C:\projects\octokit-net\Octokit\Http\SimpleJsonSerializer.cs:line 193 at Octokit.PocoJsonSerializerStrategy.DeserializeObject(Object value, Type type) in C:\projects\octokit-net\Octokit\SimpleJson.cs:line 1522 at Octokit.Internal.SimpleJsonSerializer.GitHubSerializerStrategy.DeserializeObject(Object value, Type type) in C:\projects\octokit-net\Octokit\Http\SimpleJsonSerializer.cs:line 193 at Octokit.SimpleJson.DeserializeObject(String json, Type type, IJsonSerializerStrategy jsonSerializerStrategy) in C:\projects\octokit-net\Octokit\SimpleJson.cs:line 591 at Octokit.SimpleJson.DeserializeObject[T](String json, IJsonSerializerStrategy jsonSerializerStrategy) in C:\projects\octokit-net\Octokit\SimpleJson.cs:line 603 at Octokit.Internal.SimpleJsonSerializer.Deserialize[T](String json) in C:\projects\octokit-net\Octokit\Http\SimpleJsonSerializer.cs:line 22 at Octokit.Internal.JsonHttpPipeline.DeserializeResponse[T](IResponse response) in C:\projects\octokit-net\Octokit\Http\JsonHttpPipeline.cs:line 62 at Octokit.Connection.d__57`1.MoveNext() in C:\projects\octokit-net\Octokit\Http\Connection.cs:line 627 --- End of stack trace from previous location where exception was thrown --- at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task) at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task) at Octokit.ApiConnection.d__42`1.MoveNext() --- End of stack trace from previous location where exception was thrown --- at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task) at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task) at Octokit.ApiConnection.<>c__DisplayClass17_0`1.<b__0>d.MoveNext() --- End of stack trace from previous location where exception was thrown --- at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task) at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task) at Octokit.ApiPagination.d__0`1.MoveNext() --- End of inner exception stack trace --- at System.Threading.Tasks.Task.ThrowIfExceptional(Boolean includeTaskCanceledExceptions) at System.Threading.Tasks.Task`1.GetResultCore(Boolean waitCompletionNotification) at System.Threading.Tasks.Task`1.get_Result() at Dnn.CommunityMetrics.GitHubIssueActivity.GetUserActivity(ActivityDTO activity) at Dnn.CommunityMetrics.UserActivityController.GetUserActivity(ActivityDTO activity, DateTime date) at Dnn.CommunityMetrics.ActivityJob.ProcessActivities() ---> (Inner Exception #0) System.OverflowException: Value was either too large or too small for an Int32. at System.Convert.ToInt32(Int64 value) at System.Int64.System.IConvertible.ToInt32(IFormatProvider provider) at System.Convert.ChangeType(Object value, Type conversionType, IFormatProvider provider) at Octokit.PocoJsonSerializerStrategy.DeserializeObject(Object value, Type type) in C:\projects\octokit-net\Octokit\SimpleJson.cs:line 1442 at Octokit.Internal.SimpleJsonSerializer.GitHubSerializerStrategy.DeserializeObject(Object value, Type type) in C:\projects\octokit-net\Octokit\Http\SimpleJsonSerializer.cs:line 193 at Octokit.PocoJsonSerializerStrategy.DeserializeObject(Object value, Type type) in C:\projects\octokit-net\Octokit\SimpleJson.cs:line 1495 at Octokit.Internal.SimpleJsonSerializer.GitHubSerializerStrategy.DeserializeObject(Object value, Type type) in C:\projects\octokit-net\Octokit\Http\SimpleJsonSerializer.cs:line 193 at Octokit.PocoJsonSerializerStrategy.DeserializeObject(Object value, Type type) in C:\projects\octokit-net\Octokit\SimpleJson.cs:line 1522 at Octokit.Internal.SimpleJsonSerializer.GitHubSerializerStrategy.DeserializeObject(Object value, Type type) in C:\projects\octokit-net\Octokit\Http\SimpleJsonSerializer.cs:line 193 at Octokit.SimpleJson.DeserializeObject(String json, Type type, IJsonSerializerStrategy jsonSerializerStrategy) in C:\projects\octokit-net\Octokit\SimpleJson.cs:line 591 at Octokit.SimpleJson.DeserializeObject[T](String json, IJsonSerializerStrategy jsonSerializerStrategy) in C:\projects\octokit-net\Octokit\SimpleJson.cs:line 603 at Octokit.Internal.SimpleJsonSerializer.Deserialize[T](String json) in C:\projects\octokit-net\Octokit\Http\SimpleJsonSerializer.cs:line 22 at Octokit.Internal.JsonHttpPipeline.DeserializeResponse[T](IResponse response) in C:\projects\octokit-net\Octokit\Http\JsonHttpPipeline.cs:line 62 at Octokit.Connection.d__57`1.MoveNext() in C:\projects\octokit-net\Octokit\Http\Connection.cs:line 627 --- End of stack trace from previous location where exception was thrown --- at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task) at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task) at Octokit.ApiConnection.d__42`1.MoveNext() --- End of stack trace from previous location where exception was thrown --- at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task) at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task) at Octokit.ApiConnection.<>c__DisplayClass17_0`1.<b__0>d.MoveNext() --- End of stack trace from previous location where exception was thrown --- at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task) at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task) at Octokit.ApiPagination.d__0`1.MoveNext()<---

### PR DESCRIPTION
**Pull request by david-poindexter**
Resolves #63 

In addition, this PR updates the target framework and upgrades Octokit from 0.32.0 to 10.0.0 (yes, this project was that far behind).  As a result of the Octokit update, various build/deploy/package issues were discovered.  These are addressed in this PR.

### CODE REVIEW

File: Dnn.CommunityMetrics.dnn [hunk 1]
```diff
@@ -78,6 +78,10 @@
               <name>Dnn.CommunityMetrics.dll</name>
               <path>bin</path>
             </assembly>
+            <assembly>
+              <name>Octokit.dll</name>
```
Comment (valadas) [inline Dnn.CommunityMetrics.dnn:82]: We should add the version in the manifest for binging redirects purpose

File: build/Build.cs [hunk 2]
```diff
@@ -99,8 +100,15 @@ class Build : NukeBuild
         .DependsOn(Compile)
         .Executes(() =>
         {
-            var assembly = RootDirectory / "bin" / Configuration / $"{ProjectName}.dll";
-            CopyFileToDirectory(assembly, DnnBinDirectory, policy: FileExistsPolicy.Overwrite);
+            var files = RootDirectory.GlobFiles(
+                $"bin/Debug/*{ProjectName}*.dll",
```
Comment (valadas) [inline build/Build.cs:104]: Here we should use {Configuration} instead of debug or it will chocke when built in release mode in CI

**Review by valadas — CHANGES_REQUESTED** [review]
Looks good except the comments I posted

File: build/Build.cs (hunk 2 above)
Comment (david-poindexter) [reply build/Build.cs:104]: This is on `Deploy` target.

File: Dnn.CommunityMetrics.dnn (hunk 1 above)
Comment (david-poindexter) [reply Dnn.CommunityMetrics.dnn:82]: Pushed a new commit.

**Review by valadas — APPROVED** [review]
Awesome

**Pull request merged**